app/js/newTest.js

```diff
@@ -6,18 +6,18 @@ testForm.addEventListener('submit', async function(e) {
     if (inputLength > 0 && inputLength < 256) {
         let testData = {}
         testData.name = testForm.testName.value
-        testData = await jsonToFormData(testData)
-        const testPath = "test"
+        testData = jsonToFormData(testData)
+        let testPath = "test"
```

Comment:
i didnt mean dont use a const, i meant you dont need a variable here at all. just put `await sendData(testData, 'test')` on the next line and delete the variable